torch/utils/_config_module.py

```diff
@@ -278,6 +289,10 @@ def __getattr__(self, name: str) -> Any:
             if config.hide:
                 raise AttributeError(f"{self.__name__}.{name} does not exist")
 
+            alias_val = self._get_alias_val(name)
+            if alias_val is not None:
```

Comment:
FYI - you should use _UNSET_SENTINEL here, or weird stuff will happen if someone sets a config value to None.